### ISSUE TITLE
python3Packages.uvloop: build failure

### ISSUE THREAD

**Issue by terlar** [NixOS/nixpkgs#67492]
**Describe the bug**
I just updated to the latest master commit and now uvloop fails to build:
```
$ nix-build '<nixpkgs>' --no-build-output -A python3Packages.uvloop
these derivations will be built:
  /nix/store/rqpsvzbwhi7bxp85xlc7rfj0vimvgk9p-python3.7-uvloop-0.12.2.drv
building '/nix/store/rqpsvzbwhi7bxp85xlc7rfj0vimvgk9p-python3.7-uvloop-0.12.2.drv'...
builder for '/nix/store/rqpsvzbwhi7bxp85xlc7rfj0vimvgk9p-python3.7-uvloop-0.12.2.drv' failed with exit code 1; last 10 log lines:
    File "uvloop/loop.pyx", line 1449, in uvloop.loop.Loop.run_until_complete
      raise RuntimeError('Event loop stopped before Future completed.')
  RuntimeError: Event loop stopped before Future completed.
  
  ----------------------------------------------------------------------
  Ran 477 tests in 124.617s
  
  FAILED (errors=2, skipped=26)
  Test failed: <unittest.runner.TextTestResult run=477 errors=2 failures=0>
  error: Test failed: <unittest.runner.TextTestResult run=477 errors=2 failures=0>
error: build of '/nix/store/rqpsvzbwhi7bxp85xlc7rfj0vimvgk9p-python3.7-uvloop-0.12.2.drv' failed
```

**To Reproduce**
1. `$ nix-build '<nixpkgs>' --no-build-output -A python3Packages.uvloop`

**Expected behavior**
The build should be working.

**Additional context**
Just noticed this when I tried to build `python3Packages.sanic` from master which was working last time I updated.

**Metadata**
```
$ nix run nixpkgs.nix-info -c nix-info -m                                                                                 (3m28s)[100]
 - system: `"x86_64-linux"`
 - host os: `Linux 5.2.9, NixOS, 19.09.git.d42ae527190 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(root): `"nixos-18.09.2574.a7e559a5504"`
 - nixpkgs: `/home/terje.larsen/src/github.com/terlar/nix-config/nixpkgs`
```

Maintainer information:
```yaml
module: python3Packages.sanic
```

**Comment by terlar** [comment]
Any idea what could have changed @costrouc ?

Some more traceback:
```
======================================================================
ERROR: test_flush_before_shutdown (test_tcp.Test_UV_TCPSSL)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/uvloop-0.12.2/tests/test_tcp.py", line 2568, in test_flush_before_shutdown
    self.loop.run_until_complete(client(srv.addr))
  File "uvloop/loop.pyx", line 1451, in uvloop.loop.Loop.run_until_complete
    return future.result()
  File "/build/uvloop-0.12.2/tests/test_tcp.py", line 2552, in client
    data = await reader.readexactly(4)
  File "/nix/store/yglc8kgzmdaskyngv90y1mw96fl00pjy-python3-3.7.4/lib/python3.7/asyncio/streams.py", line 677, in readexactly
    raise IncompleteReadError(incomplete, n)
asyncio.streams.IncompleteReadError: 0 bytes read on a total of 4 expected bytes

======================================================================
ERROR: test_renegotiation (test_tcp.Test_UV_TCPSSL)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/uvloop-0.12.2/tests/test_tcp.py", line 2234, in test_renegotiation
    run(client)
  File "/build/uvloop-0.12.2/tests/test_tcp.py", line 2229, in run
    asyncio.gather(*tasks, loop=self.loop))
  File "uvloop/loop.pyx", line 1449, in uvloop.loop.Loop.run_until_complete
    raise RuntimeError('Event loop stopped before Future completed.')
RuntimeError: Event loop stopped before Future completed.

----------------------------------------------------------------------
Ran 477 tests in 85.679s
```

**Comment by d-goldin** [comment]
Seems like it broke after https://github.com/NixOS/nixpkgs/pull/22357 got merged and seems to have been addressed upstream in 0.13.0, see: https://github.com/MagicStack/uvloop/issues/248

**Comment by terlar** [comment]
Fixed with the merge of #67735
